playbooks/roles/monitoring-user-defined-projects/README.md

```diff
@@ -0,0 +1,71 @@
+Enable monitoring for user defined projects 
+===========================================
+
+This role enables monitoring services created in the user defined projects. The playbook performs the following steps to enable user defined project monitoring:
+ - Create a new namespace
+ - Deploy a sample prometheus app
+ - Create service to define the monitoring metrics
+ - Deploy a ServiceMonitor resouce object
+ - Update the cluster-monitoring-config object to enable monitoring for user defined projects
+ - Check the service pods running
+ - Grant the permissions to the external user
+ - Extracting the secret token, bearer access token and thanos querier URL
+ - Run querying command to access the monitoring metrics outside the cluster
+
+
+Requirements
+------------
+
+ - Running OCP 4.x cluster.
+
+
+Role Variables
+--------------
+
+| Variable                               | Required | Default                          |Comments                                                                                                     |          
+|----------------------------------------|----------|----------------------------------|-------------------------------------------------------------------------------------------------------------|
+| enable_user_defined_project_monitoring | yes      | false                            | Flag to be set to true to enable monitoring-user-defined-project-monitoring role execution                  |
+| user_email                             | yes      | empty list                       | User email-id to which grant access permission, the empty input will skip the step in the playbook          |
+| user_defined_app_name                  | no       | empty string                     | Name for deploying new a sample app, if empty the default app name *"prometheus-example-app"* will be set   |
```

Comment:
The name is different for `ServiceMonitor`. Please mention here.

Reply:
Defined a separate variable for ```ServiceMonitor``` and included the description in the README.md doc